examples/oauth/src/main.rs

```diff
@@ -191,26 +223,42 @@ async fn login_authorized(
         .bearer_auth(token.access_token().secret())
         .send()
         .await
-        .unwrap()
+        .context("failed in sending request to target Url")?
         .json::<User>()
         .await
-        .unwrap();
+        .context("failed to deserialize response as JSON")?;
 
     // Create a new session filled with user data
     let mut session = Session::new();
-    session.insert("user", &user_data).unwrap();
+    session
+        .insert("user", &user_data)
+        .context("failed in inserting serialized value into session")?;
 
     // Store session and get corresponding cookie
-    let cookie = store.store_session(session).await.unwrap().unwrap();
+    let cookie = match store
+        .store_session(session)
+        .await
+        .context("failed to store session")?
+    {
+        Some(cookie) => cookie,
+        None => {
+            return Err(AppError(async_session::Error::msg(
+                "unexpected error retrieving cookie value",
+            )))
+        }
+    };
```

Comment:
You can actually call `.context()` on an `Option` to turn it into a result, so this should work

```suggestion
        .context("failed to store session")?
        .context("unexpected error retrieving cookie value")?;
```